netbox_onboarding/tables.py

```diff
@@ -12,14 +12,14 @@
 limitations under the License.
 """
 import django_tables2 as tables
-from utilities.tables import BaseTable, ToggleColumn
+from utilities.tables import BaseTable
 from .models import OnboardingTask
 
 
 class OnboardingTaskTable(BaseTable):
     """Table for displaying OnboardingTask instances."""
 
-    pk = ToggleColumn()
+    pk = tables.LinkColumn()
```

Comment:
Does this change make it so that we can no longer select rows in the table to bulk-delete?

Reply:
Thanks! Fixed